webapp/public/js/domjudge.js

```diff
@@ -835,9 +835,15 @@ function initializeKeyboardShortcuts() {
             return;
         }
         // Check if the user is not typing in an input field.
-        if (e.target.tagName === 'INPUT' || e.target.tagName === 'TEXTAREA') {
+        if (
+            e.target.tagName === 'INPUT' || 
+            e.target.tagName === 'TEXTAREA' || 
+            e.target.classList.contains('codex-editor') || 
```

Comment:
can you add a comment and a link to the source of od editorjs, where this class is being set?